blender/arm/material/make_voxel.py

```diff
@@ -1,29 +1,251 @@
 import bpy
-
 import arm.utils
 import arm.assets as assets
+import arm.material.cycles as cycles
 import arm.material.mat_state as mat_state
-
-if arm.is_reload(__name__):
-    arm.utils = arm.reload_module(arm.utils)
-    assets = arm.reload_module(assets)
-    mat_state = arm.reload_module(mat_state)
-else:
-    arm.enable_reload(__name__)
-
+import arm.material.mat_utils as mat_utils
+import arm.material.make_particle as make_particle
+import arm.make_state as state
 
 def make(context_id):
     rpdat = arm.utils.get_rp()
-    con = make_ao(context_id)
+    if rpdat.rp_gi == 'Voxel GI':
+        con = make_gi(context_id)
+    else:
+        con = make_ao(context_id)
 
     assets.vs_equal(con, assets.shader_cons['voxel_vert'])
-    assets.fs_equal(con, assets.shader_cons['voxel_frag'])
-    assets.gs_equal(con, assets.shader_cons['voxel_geom'])
+    assets.gs_equal(con, assets.shader_cons['voxel_frag'])
+    assets.fs_equal(con, assets.shader_cons['voxel_geom'])

```

Comment:
Is this a typo?

Reply:
Oh thanks ! I would never have spotted that.